lib/offline/manifest_converter.js

```diff
@@ -205,6 +205,7 @@ shaka.offline.ManifestConverter = class {
       label: streamDB.label || null,
       type: streamDB.contentType,
       primary: streamDB.primary,
+      pixelAspectRatio: streamDB.pixelAspectRatio || null,
```

Comment:
You should add this parameter to the StreamDB extern (see `externs/shaka/offline.js`).
You should also set it in the `createStream_` method in `lib/offline/storage.js`, so it will actually be stored.

I think the Closure compiler didn't catch this problem because of the `|| null`, annoyingly.

Reply:
Fixed!